docs/tenancy/1.x/affects-mails.md

```diff
@@ -66,8 +66,9 @@ class ConfigureTenantMails
     {
         if($tenant = $event->event->tenant)
         {
-            $event->setFrom([$tenant->email_from => $tenant->email_name]);
+            $event->alwaysFrom($tenant->mail_from, $tenant->mail_from_name);
+            $event->alwaysReplyTo($tenant->mail_replyto, $tenant->mail_replyto_name);
```

Comment:
This is not needed, please add a comment above it for the user that this is only needed if they want it to operate in a certain way